app/models/event.rb

```diff
@@ -69,11 +69,11 @@ class Event < ActiveRecord::Base
   end
 
   ##
-  # Checkes if the event has a start_time and a room
+  # Checkes if the event is confirmed and has a start_time and a room
   # ====Returns
   # * +true+ or +false+
   def scheduled?
-    room && start_time ? true : false
+    room && start_time && (state == 'confirmed') ? true : false
```

Comment:
I think so, because if I eliminate it and for example the `room` is `nil` it will returns `nil` instead of `false`.

I could write that:

`room.present? && start_time.present? && (state == 'confirmed')`

But I don't know if it is an improvement.